pennylane/__init__.py

```diff
@@ -348,7 +348,7 @@ def __getattr__(name):
     if name == "grouping":
         warnings.warn(
             "The qml.grouping module is deprecated, please use qml.pauli instead.",
-            DeprecationWarning,
+            UserWarning,
```

Comment:
Is there a corresponding test that needs to be changed?

Reply:
Good question, a test was not added due to some random CI failures. We instead have pragma no cover tag.